# origin: Debian (herbert)
# cset: n/a
# inclusion: not submitted (not suitable?)
# description: fix compile error on sparc64/alpha
# revision date: 2004-09-03

diff -urN kernel-source-2.4.26/drivers/scsi/dpt_i3o.c kernel-source-2.4.26-1/drivers/scsi/dpt_i2o.c
--- kernel-source-2.4.26/drivers/scsi/dpt_i2o.c	2003-06-14 00:51:36.000000000 +1000
+++ kernel-source-2.4.26-1/drivers/scsi/dpt_i2o.c	2003-06-01 20:43:49.000000000 +1000
@@ -81,9 +81,9 @@
 #elif defined(__ia64__)
 	PROC_INTEL, PROC_IA64,
 #elif defined(__sparc__)
-	PROC_ULTRASPARC,
+	PROC_ULTRASPARC, PROC_ULTRASPARC,
 #elif defined(__alpha__)
-	PROC_ALPHA ,
+	PROC_ALPHA, PROC_ALPHA,
 #else
 	(-1),(-1)
 #endif


